documentapi/src/main/java/com/yahoo/documentapi/Parameters.java

```diff
@@ -9,5 +13,12 @@
  * @author bratseth
  */
 public class Parameters {
-    // empty
+    ThrottlePolicy throttlePolicyOverride;
```

Comment:
I suggest removing "override" from the names. It doesn't add anything.

Reply:
Did so, and that revealed use in subclass VisitorParameters.
I am not sure about binary compatibility. Does moving an interface to parent brake binary compatibility ?

Reply:
No, that is a compatible change.